js/battle.js

```diff
@@ -3032,6 +3032,13 @@ var Battle = (function () {
 		this.turnsSinceMoved = 0;
 		this.acceleration = (this.messageFadeTime < 150 ? 2 : 1);
 	};
+	Battle.prototype.updateToxicTurns = function () {
+		for (var side in this.sides) {
```

Comment:
shouldn't these be `const...of` loops?

Reply:
Yes

Reply:
...no! This is the client! What are you thinking!

Reply:
Also `this.sides` is an array, isn't it? You need to iterate it manually...

Reply:
Oh lol I didn’t realize this was client yeah that’s mb

Reply:
This is why you don’t go on github at 4 in the morning

Reply:
Oops lol

Reply:
Fixed the array iterations and the newline.